scripts/node/validator.sh

```diff
@@ -61,8 +61,8 @@ if [ "$REMME_START_MODE" = "genesis" ]; then
     echo "Writing batch injector settings..."
     sawset proposal create \
         -k /etc/sawtooth/keys/validator.priv \
-        sawtooth.validator.batch_injectors=block_info \
-        "sawtooth.validator.block_validation_rules=NofX:1,block_info;XatY:block_info,0;local:0" \
+        "sawtooth.validator.batch_injectors=block_info;obligatory_payment" \
```

Comment:
1. comma should be used, not semicolon
2. we will create our own BatchInjector for all transactions that should be enforced